### PR TITLE
[luhn] Update canonical-data.json

### PR DESCRIPTION
**Pull request by bobahop**
See https://github.com/exercism/rust/pull/1228

### CODE REVIEW

**Comment by bobahop** [comment]
Not seeing the schema problem with the latest commit. What am I missing?

**Review by SleeplessByte — COMMENTED** [review]
Hiya! 

It needs to be a UUID v4. Here is one: `8b72ad26-c8be-49a2-b99c-bcc3bf631b33`

File: exercises/luhn/canonical-data.json [hunk 1]
```diff
@@ -170,6 +170,15 @@
         "value": ":9"
       },
       "expected": false
-    }
+    },
+    {
+      "uuid": "401a68ba-98d4-11eb-a8b3-0242ac130003",
```
Comment (SleeplessByte) [inline exercises/luhn/canonical-data.json:175]:
```suggestion
      "uuid": "8b72ad26-c8be-49a2-b99c-bcc3bf631b33",
```

Comment (bobahop) [reply exercises/luhn/canonical-data.json]: I had a one-out-of-two chance and I chose wrong. Thanks! I'll remember that!

Comment (SleeplessByte) [reply exercises/luhn/canonical-data.json]: We used to allow _all_ uuids, but made this restriction so they are really always just completely random.

**Review by SleeplessByte — APPROVED** [review]

**Review by ErikSchierboom — APPROVED** [review]

**Review by rpottsoh — APPROVED** [review]

**Pull request merged**